projects/storefrontlib/shared/components/chat-messaging/avatar/avatar.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input } from '@angular/core';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { MessageEvent } from '../messaging';
+
+export const FROM_CUSTOMER = 'C';
+
+@Component({
+  selector: 'cx-avatar',
+  templateUrl: './avatar.component.html',
+})
+export class AvatarComponent {
+  @Input() message: MessageEvent;
+  iconTypes = ICON_TYPE;
+
+  constructor() {}
+
+  getInitials(author: string) {
+    return author
+      ? author
+          ?.split(' ')
```

Comment:
We no longer need it to be optional, since it is being validated in the first part
```suggestion
          .split(' ')
```